app/models/article.rb

```diff
@@ -535,9 +546,11 @@ def canonical_url_must_not_have_spaces
     errors.add(:canonical_url, "must not have spaces") if canonical_url.to_s.match?(/[[:space:]]/)
   end
 
+  # TODO: (Alex Smith) refactor liquid tag permissions
+  #
   # Admin only beta tags etc.
   def validate_liquid_tag_permissions
-    errors.add(:body_markdown, "must only use permitted tags") if liquid_tags_used.include?(PollTag) && !(user.has_role?(:super_admin) || user.has_role?(:admin))
+    errors.add(:body_markdown, "must only use permitted tags") if (liquid_tags_used & RESTRICTED_LIQUID_TAGS).any? && !(user.has_role?(:super_admin) || user.has_role?(:admin))
```

Comment:
Handling permissions, roles, and liquid tags are going to change entirely in a separate PR. But, because this PR introduces the liquid tag now by including `app/liquid_tags/email_subscription_tag.rb`, I need to somehow restrict it so users don't start messing with it, though technically it wouldn't render anything since there's no view included.

I thought of just restricting the endpoint to `admin` roles as well. But I've coordinated with the design team and we felt it would be easier for me to push a very bare-bones UI with all the logic in place (future PR) that they can then put all their design polish on. Having it restricted this way will allow our internal team to access the tag in dev (assigning their user `super_admin` locally) while still keeping it "hidden" in prod (very few people have that role)🙃 .